lua/gopher/struct_tags.lua

```diff
@@ -116,4 +116,15 @@ function M.remove(...)
   modify(unpack(cmd_args))
 end
 
+---clear all tags to struct under cursor
+function M.clear()
+  if #arg == nil or arg == "" then
+    arg = { "json" }
+  end
```

Comment:
as i fast googled `--clear-tags` option, it doesn't require any options, this could be deleted.

or if i am wrong, the function doesn't get any arguments. you might copy-past it from other function :smiley: